lib/tokenlist.cpp

```diff
@@ -485,6 +485,10 @@ static Token * skipDecl(Token *tok)
         } else if (Token::Match(vartok, "%var% [:=(]")) {
             return vartok;
         }
+        else if (Token::Match(vartok, "decltype ( auto ) ")) {
```

Comment:
This should also be `Token::simpleMatch(vartok, "decltype ( auto )")` without the extra space in the match string. Also, `simpleMatch` is used since its just string matching each token.

Reply:
fixed with `` 891440f ``